starter-dev-backend/handler.js

```diff
@@ -26,10 +27,16 @@ app.use(express.json());
 
 app.get('/', (req, res, next) => {
   return res.status(200).json({
-    message: 'Hello from root!',
+    message: 'Welcome to the starter kit!',
   });
 });
 
+app.get('/hello', (req, res) => {
+  res.send(`Hello, ${req.query.greeting}`);
+});
+
+app.use('/', routes);
```

Comment:
```suggestion
app.use('/graphql', routes);
```

^ Highly recommend this change.